types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+import { IntrinsicElementAttributes } from 'vue-tsx-support/types/dom'
+
+declare module 'vue-emotion' {
+  const styled: IntrinsicElementAttributes
+  export default styled
+  export const css: any
+  export const injectGlobal: any
+}
```

Comment:
1. you can `export * from 'emotion'` here too
2. seems no need to wrap inside `declare module 'vue-emotion'`
3. `styled` does not always support `.div` `.a` methods unless you're using babel, not sure if we should include a definition for this. And I don't think `IntrinsicElementAttributes` is the correct type 🤔

Reply:
Yes, we should make a change based on https://github.com/emotion-js/emotion/blob/master/packages/react-emotion/types/index.d.ts